CMakeLists.txt

```diff
@@ -29,10 +29,14 @@ add_executable(${PROJECT_NAME} WIN32
 
 set(CMAKE_C_FLAGS_DEBUG "${CMAKE_C_FLAGS_DEBUG} -DDEBUG")
 
-set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} /guard:cf /Qspectre")
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} /guard:cf")
+if (NOT ASAN)
```

Comment:
If you're using an option/variable here, you should declare it to get proper documentation. But I'm not sure a variable is the correct approach here? In OpenVPN 3 we used a build type instead for ASAN. Any reason not to do that?

Reply:
I see, but `CMAKE_BUILD_TYPE` is not used with MSVC generator. Looks like I need to look into CMAKE_CONFIGURATION_TYPES.